site/_scss/blocks/_heading-link.scss

```diff
@@ -1,5 +1,30 @@
 .heading-link {
```

Comment:
On issue with the current implement implementation is that mousing over the space where the anchor # should be does not cause it to appear. Since the invisible element is the part that you must click or copy, this makes it a bit trickier than necessary to get the link.

https://user-images.githubusercontent.com/1047763/128947797-808e966d-ec90-428c-80fe-c1c3b3582cf4.mov

The most straightforward way address this is to use opacity instead of visibility, but I assume we're using visibility and a pseudo element to prevent the # from being read aloud by screen readers.

Alternatively, we could use a negative margin and equal positive padding on the header to make the area to the left of the header react to hovers.

https://user-images.githubusercontent.com/1047763/128948453-b5502713-ab90-490e-bf1a-3f961fb11203.mov